modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -409,6 +409,50 @@
 #define NRFX_SPIS3_ENABLED 1
 #endif
 
```

Comment:
Drop these blank lines, please. We don't have them for other peripherals, like SPIMs or TWIMs.